content/docs/gke/troubleshooting-gke.md

```diff
@@ -417,3 +417,53 @@ kubectl -n kubeflow create secret generic kubeflow-oauth \
        --from-literal=client_id=${CLIENT_ID} \
        --from-literal=client_secret=${CLIENT_SECRET}
 ```
+
+## Managed certificates can fail if service management API not enabled
+
+This section describes how to enable service management API to avoid managed certificates failure.
+
+To check your certificate: 
+1. Run the following command:
+
+```
+kubectl -n istio-system describe managedcertificate gke-certificate
+```
+
+Make sure the certificate status is either `Active` or `Provisioning` which means it is not ready. For more details on certificate status, refer to the [certificate statuses descriptions](https://cloud.google.com/load-balancing/docs/ssl-certificates?hl=en_US&_ga=2.164380342.-821786221.1568995229#certificate-resource-status) section. Also, make sure the domain name is correct.
+
+2. Run the following command using the certificate name from the previous step:
+
+```
+gcloud beta --project=${PROJECT} compute ssl-certificates describe --global ${CERTIFICATE_NAME}
+```
+
+3. Run the following command:
+
+```
+kubectl -n istio-system get ingress envoy-ingress -o yaml
+``` 
+
+Make sure of the following: 
+** `networking.gke.io/managed-certificates` annotation value points to the name of the Kubernetes managed certificate resource and is `gke-certificate`;
```

Comment:
The formatting is broken here on the rendered page. Tip: Use indentation rather than double asterisks to create a sub-list. See the example on this page:
https://www.kubeflow.org/docs/gke/deploy/deploy-cli/#prepare-your-environment

Source for the page:
https://raw.githubusercontent.com/kubeflow/website/master/content/docs/gke/deploy/deploy-cli.md

Reply:
LGTM
I fixed it to conform to other Kubeflow pages formatting rules.